app/helpers/dfe_sign_in/user_helper.rb

```diff
@@ -1,16 +1,18 @@
 module DfeSignIn
   module UserHelper
-    def user_details(user)
-      user.full_name.presence || unknown_user(user)
+    def user_details(user, options = {})
```

Comment:
Would this be better as a single named parameter instead of an options hash?

Reply:
Good call, that actually makes cycling down to that private method work a bit better too